test/integration-test-main.sh

```diff
@@ -1445,7 +1445,14 @@ function add_all_tests {
     add_tests test_external_modification
     add_tests test_read_external_object
     add_tests test_update_metadata_external_small_object
-    add_tests test_update_metadata_external_large_object
+    if ! uname | grep -q Darwin; then
+        # [NOTE]
+        # This test is very time consuming on OSX and will not run.
+        # And this test should be no different between OSX and
+        # other OSs. so skip this test on OSX.
+        #
+        add_tests test_update_metadata_external_large_object
```

Comment:
Why is this only slow on macOS?

Reply:
Not only is this process heavy, but it probably runs slower on macOS than on other Linux.
(It seems that it is a difference in the specifications of Runner's HOST of Github Actions.)
I wanted to reduce the latency in the case of repeated builds, so I thought about bypassing this test that I added last.
But, even if we bypass this part, it will take longer than Linux.
If the test should pass equally on all operating systems, this PR will be withdrawn.

Reply:
Strange but let's investigate this more later.